modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -131,6 +149,12 @@ const Modal = ({
 		return <div className="liferay-modal-body" ref={bodyRef}></div>;
 	};
 
+	const ariaLabelledby = useMemo(() => {
+		modalLabelIdCounter++;
+
+		return `open-modal-label-${modalLabelIdCounter}`;
+	}, []);
+
```

Comment:
@diegonvs The same as https://github.com/liferay-frontend/liferay-portal/pull/1988#discussion_r832012284

>Do we need this, and the related `ariaLabelledby` logic? It seems like we are duplicating what will be added to all `ClayModal`s in the next Clay iteration, with https://github.com/liferay/clay/pull/4737. Clay update: https://github.com/liferay-frontend/liferay-portal/pull/2029

Reply:
Removing...

Reply:
Updated! Thank you!